src/test/mir-opt/while-storage.rs

```diff
@@ -5,7 +5,7 @@ fn get_bool(c: bool) -> bool {
     c
 }
 
-// EMIT_MIR rustc.while_loop.PreCodegen.after.mir
+// EMIT_MIR while_storage.while_loop.PreCodegen.after.mir
```

Comment:
do you think it's doable with reasonable effort to not have to specify the prefix and instead have the test suite prepend the file name (without `.rs`) automatically?

Reply:
if i can find the function in rustc which turns a filename into a cratename, it should be possible (note the file is `while-storage` but the crate is `while_storage`)

Reply:
oh actually, I don't think it's super easily possible, because sometimes the expectations are from `core` and other crates... which we can't predict easily.

Reply:
jup makes sense, let's just go with the status quo